android/src/main/kotlin/io/customer/customer_io/CustomerIoPlugin.kt

```diff
@@ -19,7 +20,7 @@ import io.flutter.plugin.common.MethodChannel.Result
  * Android implementation of plugin that will let Flutter developers to
  * interact with a Android platform
  * */
-class CustomerIOPlugin : FlutterPlugin, MethodCallHandler {
+class CustomerIoPlugin : FlutterPlugin, MethodCallHandler {
```

Comment:
Had to revert back to `CustomerIoPlugin` because it's auto-generated and if you update it, it stops communicating to native bindings.